manifest.toml

```diff
@@ -0,0 +1,30 @@
+# Manifest contains metadata about the driver. To learn about the different
```

Comment:
Please, remove all comments from the template and leave only any comment you need that is specific to your manifest, or remove comments altogether.